File: leetcode/problems/go/1344. Angle Between Hands of a Clock/solve.go
package main

import "math"

func angleClock(hour int, minutes int) float64 {
	minute_angle := float64(minutes) * 6
	hour_angle := (float64(hour%12) + float64(minutes)/60) * 30
	diff := math.Abs(hour_angle - minute_angle)
	if diff > 180 {
		return 360 - diff
	}
	return diff
}

File: leetcode/problems/typescript/1344. Angle Between Hands of a Clock/solve.ts
function angleClock(hour: number, minutes: number): number {
  const minuteAngle = minutes * 6;
  const hourAngle = (hour % 12) * 30 + minutes / 2;
  const diff = Math.abs(hourAngle - minuteAngle);
  return Math.min(diff, 360 - diff);
}
